### PR TITLE
input: fix keybind command

### PR DESCRIPTION
**Pull request by na-na-hi**
Fixes various issues with the `keybind` command.

### CODE REVIEW

File: DOCS/man/input.rst [hunk 1]
```diff
@@ -925,9 +925,9 @@ Remember to quote string arguments in input.conf (see `Flat command syntax`_).
     only be set on the key specified by ``name``.
 
 ``keybind <name> <command>``
```
Comment (kasper93) [inline DOCS/man/input.rst]: change it also here

File: player/command.c [hunk 2]
```diff
@@ -6633,7 +6633,10 @@ static void cmd_key_bind(void *p)
         return;
     }
     const char *target_cmd = cmd->args[1].v.s;
-    mp_input_bind_key(mpctx->input, code, bstr0(target_cmd));
+    const char *comment = cmd->args[2].v.s;
+    if (comment && !*comment)
```
Comment (kasper93) [inline player/command.c:6632]: we generally do `comment[0]` check.

File: player/command.c [hunk 3]
```diff
@@ -6626,17 +6626,15 @@ static void cmd_key_bind(void *p)
...
-        return;
-    }
+    const char *key = cmd->args[0].v.s;
     const char *target_cmd = cmd->args[1].v.s;
     const char *comment = cmd->args[2].v.s;
     if (comment && !*comment)
         comment = NULL;
-    mp_input_bind_key(mpctx->input, code, bstr0(target_cmd), comment);
+    if (!mp_input_bind_key(mpctx->input, key, bstr0(target_cmd), comment)) {
+        MP_ERR(mpctx, "%s is not a valid input name.\n", key);
```
Comment (kasper93) [inline player/command.c:6635]:
``` suggestion
        MP_ERR(mpctx, "'%s' is not a valid input name.\n", key);
```

**Comment by avih** [comment]
Nice PR.

File: DOCS/man/input.rst (hunk 1 above)
Comment (kasper93) [reply DOCS/man/input.rst]: Wouldn't it make sense to do it in this commit? I don't want to be this pedantic, so it is fine, just going through commits it wasn't visible.

**Review by kasper93 — APPROVED** [review]

**Pull request merged**